Header header
ar_sys/ArPixel[] pixel_location
geometry_msgs/Point[] global_position
